cloud/amazon/ec2_eip.py

```diff
@@ -58,6 +58,12 @@
     required: false
     default: false
     version_added: "2.0"
+  private_ip_address:
```

Comment:
@wimnat Can we keep public_ip and private_ip naming consistent? Either change this to private_ip or change public_ip to public_ip_address?